The inferno project is licensed under the CDDL, so patching that out.

diff --git a/Cargo.toml b/Cargo.toml
index dec956b..b27bab3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,12 +37,6 @@ optional = true
 [dependencies.findshlibs]
 version = "0.10"
 
-[dependencies.inferno]
-version = "0.11"
-features = ["nameattr"]
-optional = true
-default-features = false
-
 [dependencies.libc]
 version = "^0.2.66"
 
@@ -111,7 +105,6 @@ optional = true
 _protobuf = []
 cpp = ["symbolic-demangle/cpp"]
 default = ["cpp"]
-flamegraph = ["inferno"]
 frame-pointer = []
 prost-codec = [
     "prost",
